src/include/osl_pvt.h

```diff
@@ -1029,10 +1056,12 @@ class Symbol {
     unsigned m_symtype : 4;     ///< Kind of symbol (param, local, etc.)
     unsigned m_has_derivs : 1;  ///< Step to derivs (0 == has no derivs)
     unsigned m_const_initializer : 1;  ///< initializer is a constant expression
-    unsigned m_connected_down : 1;   ///< Connected to a later/downstream layer
-    unsigned m_initialized : 1;      ///< If a param, has it been initialized?
-    unsigned m_lockgeom : 1;         ///< Is the param not overridden by geom?
-    unsigned m_allowconnect : 1;     ///< Is the param not overridden by geom?
+    unsigned m_connected_down : 1;  ///< Connected to a later/downstream layer
+    unsigned m_initialized : 1;     ///< If a param, has it been initialized?
+    unsigned m_interpolated : 1;    ///< Is the param overridden by geom?
+    unsigned m_interactive : 1;     ///< May the param change interactively?
+    unsigned m_noninteractive : 1;  ///< The param def won't modify interactively
```

Comment:
Not sure I understand why we need interactive and noninteractive flags.

Reply:
In discussion with Lee, we wanted leave room for the source code hints to be able to explicitly say `[[ int interactive = 0 ]]` to indicting being quite sure that this parameter is not one that lookdev artists will need to adjust interactively, whereas lacking any hint one way or the other has the possibility for a runtime control to allow everything not marked to be flipped to interactive.

It's still a half-baked idea that won't be fleshed out until we have a UI and understand more how it's used in practice, but I'm leaving as a placeholder so that the hint is really 3-state on, off, or not specified.